### PR TITLE
#19 Added extraction of page title

### PR DESCRIPTION
**Pull request by antoniaklja**
Added initial version of page title extraction.

### CODE REVIEW

**Comment by antoniaklja** [comment]
I can prepare initial version of this module if you wish.

File: protocol/http_scanner.go [hunk 1]
```diff
@@ -36,9 +36,18 @@ func (hps * HTTPProtocolScanner) ScanProtocol(hiddenService string, proxyAddress
 			return
 		}
 
+		// Reading all http headers
+		log.Printf("HTTP response headers: %s\n", report.ServerVersion)
```
Comment (s-rah) [inline protocol/http_scanner.go]: We shouldn't print out server version here. This line could probably be removed.

**Comment by s-rah** [comment]
Thanks, this looks good - minor comment on the output.

**Comment by antoniaklja** [comment]
Done, I left line which set server version.

**Comment by s-rah** [comment]
Merged in d83d10bd93c9d8d74d438134af672b28cda630b3. Thanks!

**Pull request closed without merging**